api/camunda/camunda-openapi.yaml

```diff
@@ -1818,6 +2167,9 @@ components:
         id:
           type: "integer"
           format: "int64"
+        key:
+          type: "integer"
+          format: "int64"
```

Comment:
category: missing descriptions, owner: API author

This entire schema looks like it's missing descriptions; I think I might have tagged someone on this in a previous spec update.

Reply:
Yup, nudged in https://github.com/camunda/camunda-docs/pull/4217#discussion_r1746211079.